common/main/cpp/native_glue.cc

```diff
@@ -237,6 +237,20 @@ namespace litecore {
             }
         }
 
+        jstringSlice::jstringSlice(JNIEnv *env, jbyteArray jchars) {
+            if (jchars == nullptr) {
+                _slice = kFLSliceNull;
+            } else {
+                size_t length = env->GetArrayLength(jchars);
+                if (length <= 0) {
+                    _slice = kFLSliceNull;
+                } else {
+                    auto *chars = env->GetByteArrayElements(jchars, nullptr);
```

Comment:
char* needs be released after use. `jbyteArraySlice` does handle that.

Reply:
Woo.... Thanks Pasin!

Reply:
Fixed.  Actually, you fixed it...